test/blackbox-tests/test-cases/pkg/pin-depends.t

```diff
@@ -175,10 +175,18 @@ Pin to an invalid opam file
   unsupported or missing file format version; should be 2.0 or older
   [1]
 
-Pin to an HTTP archive doesn't work
+Pin to an HTTP archive work
 
-  $ runtest "http://0.0.0.0/tarball.tgz"
-  File "foo.opam", line 1, characters 0-0:
-  Error: Could not determine location of repository http://0.0.0.0/tarball.tgz
-  Hint: Specify either a file path or git repo via SSH/HTTPS
-  [1]
+  $ mkdir _source/
+  $ cat > _source/bar.opam << EOF
+  > opam-version: "2-0"
+  > EOF
+  $ tar -czf tarball.tgz -C _source bar.opam
```

Comment:
Use just `cf` then it creates a tarball without compressing. Saves a few cycles in compression and decompression.